apps/src/templates/manageStudents/ConfirmRemoveStudentDialog.jsx

```diff
@@ -76,9 +76,6 @@ export default class ConfirmRemoveStudentDialog extends React.Component {
                     style={styles.sendHomeInstructionsButton}
                     tabIndex="1"
                   />
-                  <p>
```

Comment:
Is it intentional and desired that "Give them at least a few days to follow these instructions before you remove them." was moved above the button?

Reply:
Good question.  I checked with Brad and got the thumbs up!